Soccer Match.rtf

```diff
@@ -7,6 +7,11 @@
 
 \f0\fs24 \cf0 The app we\'92re building is soccer game. we need 2 teams of 11 players. they will play in a match. the match will keep the score. at the end we will display who wins.\
 \
+struct Team \{\
+var players: [Player(name: string, goals: Int, country: string, age: Int)]\
```

Comment:
In swift, the var type is preceded by a colon, like this: 

```
var player: Player 
```

Sometimes you can give a variable an explicit value, but we probably shouldn’t do that right now since we don’t know what players or teams we want. Also in swift, var types are always capitalized (ie, not “string” but “String”. Not int but “Int”)

Reply:
For an array of players, the var type gets brackets around it. Let’s give the Team a variable of 
```
var players: [Player]
```

Reply:
For this specific use case i think a struct is fine.